### ISSUE TITLE
installing hub on windows

### ISSUE THREAD

**Issue by maxblache** [dunglas/mercure#76]
Hi,

thanks for your work, it seems really great and to be very usefull.

I just need to make it work...

I'm trying to test it in local env on a symfony 3.4 app. I'm working on windows 10 with wamp64.

I get some trouble to run the mercure.exe and i don't find any example (evrything is just on linux), I've donwloaded the V0.4.1 x86_64 for windows, unzipped, put it on my wamp64/www folder and trying to run the exe like this : 
start mercure JWT_KEY='aVerySecretKey' ADDR='localhost:3000' ALLOW_ANONYMOUS=1 CORS_ALLOWED_ORIGINS=*

I have something like a window popping up but no message that the application is running and when I'm going to localhost:3000 no mercure welcome page

I'm missing something ??

Excuse if my question is stupid but i'm pretty beginer in this world :)

**Comment by maxblache** [comment]
ok, I finally found. 

I set the variables JWT_KEY, ADDR and ALLOW_ANONYMOUS in the windows control panel (define environment variables) and just run the mercure.exe in the terminal !

**Comment by waldifubu** [comment]
Maybe it helps:
I created a batchfile with this content
set JWT_KEY='aVerySecretKey' 
set ADDR='localhost:3000' 
set ALLOW_ANONYMOUS=1 
set CORS_ALLOWED_ORIGINS=*
mercure

All variables live only in this scope.
Maybe you can tell me, does Mercure have a gui or CLI?

**Comment by dunglas** [comment]
Hi. Mercure has no CLI, but it has a debug web GUI that you can enable by setting the DEMO env bar to 1 (be sure that the static JS and CSS files are on the same directory).

Regarding how to run it on Windows, could you add a section in the readme? Thanks!

**Comment by GrapsZ** [comment]
i've the same problem. How can i start mercure on windows 10 ?

**Comment by dunglas** [comment]
Hi @GrapsZ, have you tried the batch file suggested by @waldifubu?

**Comment by thehatami** [comment]
hi, where is release page. i cant find it.

**Comment by dunglas** [comment]
@thehatami here is it: https://github.com/dunglas/mercure/releases

**Comment by simon6023** [comment]
Hi everyone,
I also went with @waldifubu 's solution and used a batch file, however I had to remove the quotes surrounding the environment variables, they seem to provoque a `getaddrinfow` error when starting Mercure.

My final batch file looks like this :

`set JWT_KEY=YourJwtKey`
`set ADDR=localhost:3000` 
`set ALLOW_ANONYMOUS=1`
`set CORS_ALLOWED_ORIGINS=*`
`.\mercure.exe`

I'm running Windows 64bits for development, and version 0.5.1 of Mercure.

**Comment by dunglas** [comment]
@jfhovinne and I made some tests during the EU-FOSSA hackathon and everything work as expected. We tested with Windows 10 and Windows 7.

In #167, I summarized our finding in the documentation, at a glance:

* environment variables must be properly set
* `ADDR=:3000` **doesn't work**, `ADDR=localhost:3000` must be used instead
* `mercure.exe` must be allowed in Windows Defender (for public **and** private networks)

Closing this issue.
